src/main.rs

```diff
@@ -40,10 +40,10 @@ fn player_level_up(
 
 fn main() {
     App::new()
-        .insert_resource(WgpuSettings{backends: None,..default()})
         .insert_resource(GameState {counter: 10})
-        .add_plugins(DefaultPlugins)
-        .add_stage("AI",SystemStage::parallel())
-        .add_stage("SRS",SystemStage::parallel());
+        .add_plugins(DefaultPlugins
+            .set(
+                RenderPlugin { wgpu_settings: WgpuSettings { backends: None,..default() } }
```

Comment:
registration method changed in 0.10, see https://github.com/bevyengine/bevy/pull/6946